plugins/cairo-lang-macro-attributes/src/lib.rs

```diff
@@ -170,3 +170,30 @@ pub fn executable_attribute(input: TokenStream) -> TokenStream {
     };
     TokenStream::from(expanded)
 }
+
+const NO_OP_ATTR_PREFIX: &str = "__no_op_attr_";
+
+#[proc_macro]
+pub fn no_op_attribute(input: TokenStream) -> TokenStream {
```

Comment:
Maybe we could create some common helper for this and `executable_attribute`, as they only differ by prefix?

Reply:
Sure 👍🏻